scripts/run_backend_tests.py

```diff
@@ -603,7 +603,39 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed_args.generate_coverage_report:
         subprocess.check_call([sys.executable, '-m', 'coverage', 'combine'])
         report_stdout, coverage = check_coverage(True)
+        print('')
+        print(
+            '+----------------------------------------------------------------+'
+            )
+        print(
+            '|-------- '
+            'SUMMARY OF THE FILES WITH INCOMPLETE COVERAGE '
+            '---------|'
+            )
+        print(
+            '+----------------------------------------------------------------+'
+            )
+        print('')
         print(report_stdout)
+        if coverage != 100:
+            print(
```

Comment:
@Ykumar1415 One more request, sorry. Could you please write the message like this:

```
print('WARNING: Backend test coverage is below 100%.')
print('')

print('The rightmost "Missing" column above shows lines that are still uncovered.')
print('Please add tests for scenarios that exercise those lines of code so that ')
print('there are no uncovered lines in each file.')
print('')

print('For more information, please see our backend tests wiki page:')
print('    https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports')
```

I think it would be easier to read -- take a look at the output produced and see if you agree. Thanks.

Reply:
@seanlip i agree with you. These lines will be printed like : 
![image](https://github.com/oppia/oppia/assets/97088265/4eb539b7-ebd1-49f2-91d7-933387be1522)

To adhere to our code linting rule, which limits the number of characters per line, we must structure the code accordingly to achieve the same output. code : 
![image](https://github.com/oppia/oppia/assets/97088265/ddfe059f-2bdf-43af-a871-de9802d29fdd)

But output will be same. i think it is easy to read at the output. Can i update Pr ?.

Reply:
Yes that's fine (though your first screenshot doesn't show the full thing). Just note that in your print statements, the closing paren should match the indentation level of the line with the opening paren (i.e. deindent by 4 from what you have).

Reply:
@seanlip i have updated the pr. also updated screenshot in proof of changes section